qemu/tests/cfg/cpu_device_hotpluggable.cfg

```diff
@@ -1,9 +1,11 @@
 # Notes:
 #    For ppc64/ppc64le, please manually specify cpu_model in your environment
 - cpu_device_hotpluggable: install setup image_copy unattended_install.cdrom
-    only ppc64 ppc64le x86_64
+    no aarch64
     Windows:
         no WinXP, WinVista, Win7, Win8, Win10, Win2000, Win2003
+    Win2008, Win2012, s390x:
```

Comment:
Why are there these windows guests ?

Reply:
Because can not get 'ThreadCount' from these guests.
> Windows Server 2012 R2, Windows 8.1, Windows Server 2012, Windows 8, Windows Server 2008 R2, Windows 7, Windows Server 2008 and Windows Vista: This property is not supported before Windows Server 2016 and Windows 10.